Maple2.Server.Game/Session/GameSession.cs

```diff
@@ -494,6 +495,9 @@ public void DailyReset() {
         Player.Value.Account.PrestigeExp = Player.Value.Account.PrestigeCurrentExp;
         Player.Value.Account.PrestigeLevelsGained = 0;
         Send(PrestigePacket.Load(Player.Value.Account));
+        // Home
+        Player.Value.Home.DecorationRewardTimestamp = 0;
```

Comment:
Same q here. We should be resetting claimed rewards too no?

Reply:
answered above